doc/install/ubuntu-16.04.md

```diff
@@ -0,0 +1,20 @@
+# Installing cjdns on Ubuntu 16.04
+
+This is a short guide how to setup an Ubuntu cjdns box.
+
+## Install packages
+
+	apt-get install nodejs make gcc git python2
+
+## Clone, compile, install
+
+	cd /opt
+	git clone https://github.com/cjdelisle/cjdns.git
+	cd cjdns
+	./do
+	ln -s /opt/cjdns/cjdroute /usr/bin
+	(umask 077 && ./cjdroute --genconf > /etc/cjdroute.conf)
+	cp contrib/systemd/cjdns.service /etc/systemd/system/
```

Comment:
Only if the service was already loaded, and if it is, the user should know about that already and know that it might be necessary.